_posts/2023-08-25-introducing-the-new-kaskada.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Introducing The New Kaskada: Embedded in Python for  accessible Real-Time AI"
+# image: assets/images/blog/introducing-timelines-part-1.png
+author: Ryan Michael
+featured: true
+---
+
+We started Kaskada with the goal of simplifying the real-time AI/ML lifecycle, and in the past year AI has exploded in usefulness and accessibility. Generative models and Large Language Models (LLMs) have revolutionized how we approach AI. Their accessibility and incredible capabilities have made AI more valuable than it has ever been and democratized the practice of AI.
+
+Still, a challenge remains: building and managing real-time AI applications.
+
+## The Challenge of using Real-Time Data in AI Applications
+
+Real-time data for AI Applications has always been surrounded by an array of technical and infrastructural challenges. For example:
+
+1. **Infrastructure Hurdles**: Accessing real-time data often means struggling to acquire data and dealing with complex infrastructure, requiring significant time and expertise to get right.
+
+2. **Cumbersome Tools**: Traditional tools for streaming data are bulky, with steep learning curves and complex JVM-based setups.
```

Comment:
This feels duplicated -- the cumbersome tools are similar to the infrastructure. I wonder if it would be clearer to separate (a) cumbersome infrastructure from (b) data hurdles (acquiring data).